manual/C/ch_Finance-Quote.xml

```diff
@@ -195,7 +195,8 @@ SYNOPSIS
         </simpara>
       </footnote>
       can be used to fetch the current quotes of your stocks and write them directly into your
-      &app;-data file.
+      &app;-data file without starting &app; itself. Thus an automatic, 
+      regular updating of the course database is possible.
```

Comment:
course would be a new term; worth to add?
Currently the GUI of it is Price Database.